internal/gitprovider/gitprovider.go

```diff
@@ -51,6 +51,9 @@ type Interface interface {
 	// to differences in the underlying provider APIs. It is the responsibility of
 	// the caller to sort the results as needed.
 	ListPullRequests(context.Context, *ListPullRequestOptions) ([]PullRequest, error)
+
+	// GetCommitURL returns a commit URL inferred from the provided repository URL and commit SHA.
+	GetCommitURL(string, string) (string, error)
```

Comment:
```suggestion
	// GetCommitURL returns a commit URL inferred from the provided repository URL
	// and commit ID.
	GetCommitURL(repoURL string, commitID string) (string, error)
```

Reply:
Note on function signatures... with strings, ints, etc. where the type itself doesn't impart a clear understanding of what the argument represents, it is usually best to include a name for the argument, even if it is otherwise not needed. It helps a lot in understanding, in this case, what these two strings represent and which order they belong in.

Reply:
Yes makes perfect sense. I've added the names for the inputs.